### PR TITLE
translation into Brazilian Portuguese and corrections in the English …

### PR DESCRIPTION
**Pull request by luizcmarin**
…translation

Two changes are worth highlighting:
- creation of "guide" folder and translations moved there (as standard for documents)
- corrections of links and other 'little things' in English

Furthermore, all links have been verified. 🥴

### CODE REVIEW

**Review by vjik — APPROVED** [review]

**Review by samdark — CHANGES_REQUESTED** [review]
https://github.com/yiisoft/db/actions/runs/8691874638/job/23835161447?pr=819#step:6:33

**Comment by luizcmarin** [comment]
> https://github.com/yiisoft/db/actions/runs/8691874638/job/23835161447?pr=819#step:6:33

??  I didn't understand

**Comment by luizcmarin** [comment]
> https://github.com/yiisoft/db/actions/runs/8691874638/job/23835161447?pr=819#step:6:33

I don't feel comfortable dealing with this type of file. Please guide me.

"WARNINGS!
Tests: 790, Assertions: 1587, Warnings: 1, Deprecations: 3."

File: docs/guide/en/testing.md [hunk 1]
```diff
@@ -26,14 +26,10 @@ For greater ease it is recommended to use Docker containers for each DBMS, for t
 
 For running the Docker containers you can use the following command:
 
-```shell
+```dockerfile
```
Comment (samdark) [inline docs/guide/en/testing.md:29]:
```suggestion
```shell
```

**Review by samdark — APPROVED** [review]

**Pull request merged**